secrets/resource_secret_scope.go

```diff
@@ -49,13 +60,7 @@ var validScope = validation.StringMatch(regexp.MustCompile(`^[\w\.@_/-]{1,128}$`
 
 // ResourceSecretScope manages secret scopes
 func ResourceSecretScope() common.Resource {
-	s := common.StructToSchema(SecretScope{}, func(s map[string]*schema.Schema) map[string]*schema.Schema {
-		// TODO: DiffSuppressFunc for initial_manage_principal & importing
```

Comment:
I don't think this is needed because you can't update this resource (is that intentional?)

Reply:
basically any change in the configuration makes it a different resource, plus we don't have Update API...